projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/cpq-quote/cpq-quote.e2e.cy.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as quote from '../../../helpers/b2b/b2b-quote';
```

Comment:
Please, create a `quote` folder under `cpq` folder and move this cypress tests there

Reply:
Hi Larisa, I've come across some shared code from the b2b-quote function that I'm reusing. Would you recommend recreating this function, or should I keep it as it is?

Reply:
Hi Jali, please, reuse the existing methods.